CHANGES.rst

```diff
@@ -18,6 +18,7 @@ Improvements
   (:issue:`5560`, :pr:`5571`)
 - Allow raw HTML snippets when editing custom conference pages and event
   descriptions (:issue:`5584`, :pr:`5587`)
+- Add email checker for person links (:pr:`5478`)
```

Comment:
Not sure how clear that is... How about something like "Check email addresses of speakers/authors to avoid collisions and inconsistencies"?